backend/protocol_rpc/health.py

```
@@ -0,0 +1,109 @@
+
+    # Check database connectivity
+    db_status = "unknown"
+    try:
+        db_manager = get_database_manager()
+        with db_manager.engine.connect() as conn:
+            from sqlalchemy import text
+
+            conn.execute(text("SELECT 1"))
+            conn.commit()
+        db_status = "healthy"
+    except Exception as e:
```

Ruff (0.13.1)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)